platform/i18n/src/locales/es/StudyList.json

```diff
@@ -3,9 +3,9 @@
   "ClearFilters": "Limpiar filtros",
   "Description": "Descripción",
   "Empty": "vacío",
+  "Filter list to 100 studies or less to enable sorting": "Filtre la lista a 100 estudios o menos para habilitar la clasificación",
```

Comment:
There is no "NumOfStudiesHiggerThan100Message" key. Only "Filter list to 100 studies or less to enable sorting"